specification/sql/resource-manager/readme.md

```diff
@@ -4,42 +4,42 @@
 
 This is the AutoRest configuration file for Sql.
 
-
-
----
 ## Getting Started
+
 To build the SDK for Sql, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
----
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the Sql API.
 
 ``` yaml
 title: SqlManagementClient
 description: The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.
 openapi-type: arm
-tag: package-2017-03-preview
+tag: package-composite-v1

```

Comment:
I think this makes a lot of sense. It makes explicit the composite nature of input-file tags.

To be clear though, the `v1` here isn't something that we expect to be exposed to customers, and is just a hint for the SDK creator, right?

Reply:
Correct